dev/release/verify_rc.sh

```diff
@@ -34,7 +34,7 @@ set -x
 VERSION="$1"
 RC="$2"
 
-ICEBERG_DIST_BASE_URL="https://dist.apache.org/repos/dist/dev/iceberg"
+ICEBERG_DIST_BASE_URL="https://dist.apache.org/repos/dist/release/iceberg"
```

Comment:
Shouldn't this use the "downloads" location?

Reply:
```sh
import_gpg_keys() {
  if [ "${VERIFY_SIGN}" -gt 0 ]; then
    download "${ICEBERG_DIST_BASE_URL}/KEYS"
    gpg --import KEYS
  fi
}
```

Is used below

Reply:
nevermind they are the same...

Reply:
https://downloads.apache.org/iceberg/KEYS is a mirror of https://dist.apache.org/repos/dist/release/iceberg/KEYS

and might be delayed when updating